Apps/FCM/ios/FCMSampleApp/AppDelegate.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import React
+import CioMessagingPushFCM
+import CioTracking
+import Firebase
+import FirebaseMessaging
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+  
+  let moduleName: String = "FCMSampleApp"
+  let initialProps: [AnyHashable: Any] = [:]
+  var window: UIWindow?
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Add custom initialization code here
+    // This fixes an issue in which the React Native SDK identifies a profile, but an iOS device isn't added to the profile.
+    // Important to call this code before calling `registerForRemoteNotifications`.
+    CustomerIO.initialize(siteId: Env.siteId, apiKey: Env.apiKey, region: .US) { config in
+      config.autoTrackPushEvents = true
+      config.logLevel = .debug
+    }
+
+    // Configure Firebase
+    FirebaseApp.configure()
+    
+    var modifiedLaunchOptions = launchOptions
+
+    if let options = launchOptions,
```

Comment:
Can you add a comment here explaining that this is a documented workaround? Perhaps even linking to the official docs to the workaround?

Reply:
Added comment in Objective-C version.